qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/domain/TestActivitySuite.kt

```diff
@@ -30,23 +30,36 @@ import com.mapbox.navigation.qa_test_app.view.UpcomingRoadObjectsActivity
 import com.mapbox.navigation.qa_test_app.view.componentinstaller.ComponentsActivity
 import com.mapbox.navigation.qa_test_app.view.componentinstaller.ComponentsAltActivity
 import com.mapbox.navigation.qa_test_app.view.customnavview.MapboxNavigationViewCustomizedActivity
+import com.mapbox.navigation.qa_test_app.view.main.SelectDestinationDialogFragment
 import com.mapbox.navigation.qa_test_app.view.util.RouteDrawingActivity
 
-typealias LaunchActivityFun = (Activity) -> Unit
+typealias LaunchActivityFun = (AppCompatActivity) -> Unit
+
+data class Destination(val name: String, val point: Point)
 
 object TestActivitySuite {
 
-    @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
+    const val CATEGORY_NONE = "none"
+    const val CATEGORY_DROP_IN = "Drop-In UI"
+    const val CATEGORY_COMPONENTS = "Component Installer"
```

Comment:
We have a file `com.mapbox.navigation.qa_test_app.domain.Constants.kt`. We can move the constants to this file and use `CATEGORY_NONE` in `TestActivityDescription`

```
data class TestActivityDescription(
    val label: String,
    val fullDescriptionResource: Int,
    val category: String = CATEGORY_NONE
```

Reply:
Done